esp/xslt/esdl2monitor.xslt

```diff
@@ -182,22 +183,26 @@ END;
   string csndServer := '127.0.0.1' : stored('cassandraServer', FORMAT(SEQUENCE(1)));
   string csndUser := '' : stored('cassandraUser', FORMAT(SEQUENCE(2)));
   string csndPassword := '' : stored('cassandraPassword', FORMAT(PASSWORD, SEQUENCE(3)));
-  string csndKeySpace := 'eclresultmonitoring' : stored('cassandraKeyspace', FORMAT(SEQUENCE(4)));
+
+  string csndKeySpaceFrom := 'monitors_a' : stored('fromKeyspace', FORMAT(SEQUENCE(4)));
+  string csndKeySpaceTo := 'monitors_a' : stored('toKeyspace', FORMAT(SEQUENCE(4)));
 
   string monAction := 'Create' : STORED('MonAction', FORMAT(SELECT('Create,Run'), SEQUENCE(5)));
   string userId := '' : stored('UserId', FORMAT(SEQUENCE(6)));
   string service_url := '' : stored('QueryURL', FORMAT(SEQUENCE(7)));
   string service_name := '' : stored('QueryName', FORMAT(SEQUENCE(8)));
-  string MonitorIdIn := '' : stored('MonitorId', FORMAT(SEQUENCE(9)));
 
+<xsl:if test="$diffaction='Run'">
+  string MonitorIdIn := '' : stored('MonitorId', FORMAT(SEQUENCE(9)));
+</xsl:if>
   requestIn := DATASET([], the_requestLayout) : STORED ('<xsl:value-of select="$requestType"/>', FEW, FORMAT(FIELDWIDTH(100),FIELDHEIGHT(30), sequence(100)));
 
 MonSoapcall(DATASET(the_requestLayout) req) := FUNCTION
 
   // Wrap it so that request would look like:
   // <AssetReportRequest><Row><User>...</User><Options>..</Options><SearchBy>...</SearchBy></Row></AssetReportRequest>
   in_rec := record
-    dataset (the_requestLayout) <xsl:value-of select="$requestType"/> {xpath('<xsl:value-of select="$requestType"/>'), maxcount(1)};
+    dataset (the_requestLayout) <xsl:value-of select="$requestType"/> {xpath('<xsl:value-of select="$requestType"/>/Row'), maxcount(1)};
```

Comment:
Right now requestType is definitely required.  I drive the process off of the ESDL service method definition which uses method name, requestType and responseType.  I could simplify the generation of the "Compare" only function, but the real focus now is the Monitor function which needs the request to make a soapcall to the original query.